.travis.yml

```diff
@@ -79,7 +79,11 @@ install:
     fi
 
 script:
-  - if [ -n "$BUILD" ]; then cmake . -DMETEREXEC_ROOTACCESS=OFF -DENABLE_OCR=ON -DSML_HOME=/usr/local/src/libsml/sml && make && make test; fi
+  - ls
```

Comment:
thx! No clue why the previous code/script was working at all or when it stopped working. can we get this as a sep. PR and fix travis first?

Reply:
I'll do a PR with that one.

Reply:
see PR#410. I'd try next whether travis xenial can use gcc8 as well.